src/style.css

```diff
@@ -2,6 +2,44 @@
   height: 5rem;
 }
 
+.pop-up-background {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  min-height: 100vh;
+  height: 100%;
+  overflow: auto;
+  background-image: url('https://wallpapercave.com/wp/wp2311074.png');
+}
+
+.pop-up-container {
+  height: 100%;
+  border-radius: 20px;
+  flex-basis: 800px;
+  background-color: #733526;
+  border-color: #b5895a !important;
```

Comment:
Be cautious using `!important`.
Once the project gets bigger, it will get hard to find what is causing unexpected style behaviour.